haystack/pipeline.py

```diff
@@ -593,6 +595,29 @@ def run(self, **kwargs):
         return kwargs, "output_1"
 
 
+class QueryClassifier:
+        outgoing_edges = 2
+        query_vectorizer = pickle.load(
+            urllib.request.urlopen(
+                "https://raw.githubusercontent.com/shahrukhx01/ocr-test/main/query_vectorizer.pickle"
```

Comment:
I think Deepset team host these model on their s3 (@tholor WDYT?)
Also It is better to pass model via constructor

Reply:
Yep, I agree. I will upload it to our s3.

Reply:
@lalitpagaria thanks for your feedback.
@tholor could you also please upload the TF-IDF vectorizer pickle for feature extraction on S3. https://raw.githubusercontent.com/shahrukhx01/ocr-test/main/query_vectorizer.pickle

Reply:
Done: https://ext-models-haystack.s3.eu-central-1.amazonaws.com/gradboost_query_classifier/vectorizer.pickle